src/pages/GourmetList/FoodDataList.js

```diff
@@ -0,0 +1,280 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components/macro';
+
+const FoodList = () => {
+  const [list, SetList] = useState([]);
```

Comment:
```suggestion
  const [list, setList] = useState([]);
```
작명 신경써주세요~
대문자로 시작하는건 컴포넌트 이름쓸때로 기억하시면 수월할꺼에요

Reply:
네! 수정했습니다!